sdks/java/io/expansion-service/build.gradle

```
@@ -33,6 +33,8 @@ ext.summary = "Expansion service serving several Java IOs"
 dependencies {
   compile project(":sdks:java:expansion-service")
   compile project(":sdks:java:io:kafka")
+  runtime project(":sdks:java:io:jdbc")
   runtime library.java.kafka_clients
+  runtime library.java.postgres
```

nit: It might not be clear why this is here
```suggestion
  // Include postgres so it can be used with external JDBC
  runtime library.java.postgres
```

---

Done